### PR TITLE
fix: ionic push code snippets

### PR DESCRIPTION
**Pull request by psturc**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-8069

## What
* Fixed missing `import` statements in code snippets
* Added push message handling for ionic

### CODE REVIEW

File: modules/ROOT/pages/_partials/push/handling-push-notifications.adoc [hunk 1]
```diff
@@ -54,14 +54,14 @@ Cordova::
 
 NOTE: You can only develop push applications using Ionic. Plain Cordova is not supported.
 
-. Install the plugin:
-+
 [source,bash]
 ----
-$ npm install --save @ionic-native/push
+pushObject.on('notification').subscribe(data => {
```
Comment (finp) [inline modules/ROOT/pages/_partials/push/handling-push-notifications.adoc]: could we reinstate the steps, something like:
. Add the following code to your app:
<code>
. Review the documentation...

File: modules/ROOT/pages/_partials/push/handling-push-notifications.adoc [hunk 2]
```diff
@@ -54,14 +54,14 @@ Cordova::
...
+    // Handle incoming message
+});
 ----
 
-. Review the documentation at link:https://ionicframework.com/docs/native/push[Ionic Push plugin^] for details about using the plugin. 
+Review the documentation at link:https://ionicframework.com/docs/native/push[Ionic Push plugin^] for details about using the plugin.
```
Comment (finp) [inline modules/ROOT/pages/_partials/push/handling-push-notifications.adoc]: is this still correct, given that we are no longer asking users to install the plugin?

File: modules/ROOT/pages/_partials/push/handling-push-notifications.adoc (hunk 1 above)
Comment (psturc) [reply modules/ROOT/pages/_partials/push/handling-push-notifications.adoc]: agreed

File: modules/ROOT/pages/_partials/push/handling-push-notifications.adoc (hunk 2 above)
Comment (psturc) [reply modules/ROOT/pages/_partials/push/handling-push-notifications.adoc]: Installing the push plugin is a prerequisite for ionic cordova. See [import the libraries](https://deploy-preview-648--aerogear-docs-preview.netlify.com/aerogear/latest/push-notifications/#setting-up-sdk-push) section for Cordova. 
So referring to the ionic push plugin is still in place imo.

Comment (finp) [reply modules/ROOT/pages/_partials/push/handling-push-notifications.adoc]: ah, you're right, thanks

**Pull request merged**